src/main/java/com/compstore/controller/ProcessorBrandController.java

```diff
@@ -0,0 +1,55 @@
+package com.compstore.controller;
+
+import com.compstore.dto.dictionary.ProcessorBrandComboDataDTO;
+import com.compstore.dto.dictionary.ProcessorBrandCreateRequestDTO;
+import com.compstore.dto.dictionary.ProcessorBrandDTO;
+import com.compstore.service.IProcessorBrandService;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/processor-brands")
+@AllArgsConstructor
+@Tag(name = "ProcessorBrandController")
+public class ProcessorBrandController {
+
+    private final IProcessorBrandService processorBrandService;
+
+    @GetMapping
+    public ResponseEntity<List<ProcessorBrandDTO>> getAllProcessorBrands() {
+        return ResponseEntity.status(HttpStatus.OK)
+                .body(processorBrandService.getAllProcessorBrands());
```

Comment:
Dałem getAll zamiast getByUUID, gdyż pod formularzem pewnie będzie lista, a nie uważam na ten moment byśmy potrzebowali pojedynczy rekord wyświetlać. W razie czego jak wyjdzie, że trzeba to się doda